packages/pigeon/tool/run_tests.dart

```diff
@@ -31,6 +31,9 @@ const Map<String, _TestInfo> _tests = <String, _TestInfo>{
   'android_unittests': _TestInfo(
       function: _runAndroidUnitTests,
       description: 'Unit tests on generated Java code.'),
+  'android_kotlin_unittests': _TestInfo(
```

Comment:
@gaaclarke We need to add this as a default down around line 507 so that this runs default default (and thus in CI), right?

Reply:
I'd just add it to `run_tests.sh` to keep it in line with the other tests until we migrate fully everything away from there.

Reply:
Meaning set it up the way `run_dart_unittests` works in `run_test.sh`, where all the logic stays here but the call is driven from `run_tests.sh`? SGTM.

Reply:
It looks like this still needs to be done; @gaaclarke can you help with the details of wiring this up if you want to keep using run_test.sh for this?

Also, I'm not seeing any reference to Swift in run_tests.sh, which makes me think we missed this in the Swift PR. @gaaclarke Are we actually running Swift tests in CI?

Reply:
Added this logic for Dart.
Should I add this for Swift in this PR too or it's better a new PR?

Reply:
Probably best as a follow-up PR since this is already a large PR.